cases/services/get_cases.py

```diff
@@ -3,11 +3,40 @@
 
 import requests
 from django.utils import timezone
+from django.utils.text import slugify
+
 from requests.exceptions import HTTPError
 
-from cases.models import Visual
-from cases.services.clean_data import remove_provinces
+from cases.models import Visual, Country
+
+from cases.services.get_country_data import parse_country_data, get_countries_with_provinces, get_countries_without_provinces
+
+
+def bulk_insert_data(visuals):
+    vis = []
+    for v in visuals:
+        name = v.country.name
+        slug = slugify(name)
+        country, created = Country.objects.get_or_create(name=name, slug=slug)
+        vis.append(Visual(date=v.date, cases=v.cases, deaths=v.deaths, recovered=v.recovered, country=country))
+    Visual.objects.bulk_create(vis)
 
+def get_local_data(data):
```

Comment:
What does this function do?
I'm sorry, I don't understand.

Reply:
It loads the locally saved data for testing purposes instead of making a http request. The data is stored locally in a data.py file.

Reply:
Wow! Nice.